rest/taskana-rest-spring/src/test/java/pro/taskana/monitor/rest/models/ReportRepresentationModelTest.java

```diff
@@ -61,7 +63,7 @@ void testEmptyReport() {
     // meta
     ReportRepresentationModel.MetaInformation meta = resource.getMeta();
     assertEquals("WorkbasketReport", meta.getName());
-    assertEquals("2019-01-02T00:00:00Z", meta.getDate());
+    assertEquals(Instant.parse("2019-01-02T00:00:00Z"), meta.getDate());
```

Comment:
Junit assertions?

Reply:
not part of my ticket. -> Done by @tge20 in #1346